### ISSUE TITLE
Deploy to a subfolder?

### ISSUE THREAD

**Issue by ffxsam** [arunoda/meteor-up-legacy#476]
Is there a way to deploy to a subfolder, e.g. `http://apps.com/tracker`? I tried this:

```
  // Configure environment
  "env": {
    "ROOT_URL": "http://apps.com/tracker"
  },
```

But it doesn't quite work. And once I click on a link to a detail page, I would expect it to go to `http://apps.com/tracker/vELwe6rB3YbEf5grR` but instead it goes to `http://apps.com/vELwe6rB3YbEf5grR`.

Is there an elegant way to do this?

**Comment by MomchilGorchev** [comment]
Hi, this will be really valuable if possible. Better to be able to use one domain with subdomains instead of buying a new domain for every app

**Comment by nathanmarks** [comment]
@ffxsam did you figure this out?

**Comment by ffxsam** [comment]
@nathanmarks Nope, and I don't think it's possible due to how routes work. Unless there's some way to tell Iron Router that the root is `/subfolder` rather than `/`.
